helsingborg-stad/app-mitt-helsingborg
# Fix/ios appname and icons

## Explain the changes you’ve made

- Changed iOS app name from "MittHelsingborg" to "Mitt Helsingborg".
- Added new app icon.

## How to test the changes?

```
$ yarn clear-cache
$ yarn ios
```
Changing the app name can possible break things, so if it does'nt work check in your /ios folder if there are duplicated .xcworkspace and .xcodeproj files. Delete the old ones and try build again. 

## Was this feature tested in the following environments?
- [] Storybook on a iOS device/simulator.
- [] Building the Application on a Android device/simulator.

## Screenshots (optional)
![settings@3x](https://user-images.githubusercontent.com/21363149/106622679-22d61800-6574-11eb-8d94-74454f7e33ea.png)
